openbas-front/src/admin/components/simulations/simulation/injects/ExerciseInjects.tsx

```diff
@@ -201,66 +202,22 @@ const ExerciseInjects: FunctionComponent<Props> = () => {
     injectContext.onBulkDeleteInjects(injectsToProcess.map((inject: Inject) => inject.inject_id));
   };
 
-  const [openDialog, setOpenDialog] = React.useState<boolean>(false);
-  const handleCloseDialog = (
-    event?: React.SyntheticEvent | Event,
-    reason?: SnackbarCloseReason,
-  ) => {
-    if (reason === 'clickaway') {
-      return;
-    }
-    setOpenDialog(false);
-  };
-  const [detailsLink, setDetailsLink] = React.useState<string>('');
-
-  useEffect(() => {
-    if (openDialog) {
-      setTimeout(() => {
-        handleCloseDialog();
-        setDetailsLink('');
-      }, 6000);
-    }
-  }, [openDialog]);
-
   const massTestInjects = () => {
     bulkTestInjects(injectsToProcess.map((inject: Inject) => inject.inject_id)).then((result: { data: InjectTestStatus[] }) => {
       if (numberOfSelectedElements === 1) {
-        setDetailsLink(`/admin/exercises/${exercise.exercise_id}/tests/${result.data[0].status_id}`);
-        setOpenDialog(true);
+        MESSAGING$.notifySuccess(t('Inject test has been sent, you can view test logs details on {itsDedicatedPage}.', {
+          itsDedicatedPage: <Link to={`/admin/exercises/${exercise.exercise_id}/tests/${result.data[0].status_id}`}>{t('its dedicated page')}</Link>,
+        }));
       } else {
-        setDetailsLink(`/admin/exercises/${exercise.exercise_id}/tests`);
-        setOpenDialog(true);
+        MESSAGING$.notifySuccess(t('Inject test has been sent, you can view test logs details on {itsDedicatedPage}.', {
+          itsDedicatedPage: <Link to={`/admin/exercises/${exercise.exercise_id}/tests`}>{t('its dedicated page')}</Link>,
+        }));
       }
     });
   };
```

Comment:
Why this part is duplicate between exercise and scenario ? Can you use Context to avoid it ?

Reply:
To do in a later technical improvement task